util/pyclient/test_client.py

```diff
@@ -138,81 +134,60 @@ def testRetry(self):
 
     async def _testRetry(self):
         """Start servers after a delay in parallel with a write request"""
-        try:
-            async with trio.open_nursery() as nursery:
-                nursery.start_soon(self.startServersWithDelay)
-                nursery.start_soon(self.writeWithRetryAssert)
-        except:
-            raise
-        finally:
-            self.stopServers()
+        await self.writeWithRetryAssert()
 
     async def writeWithRetryAssert(self):
         """Issue a write and ensure that a retry occurs"""
-        config = self.config._replace(req_timeout_milli=5000)
-        val = 1
+        config = self.config._replace(retry_timeout_milli=0)
```

Comment:
same here - retry timeout = 0?

Reply:
Here it means that the client will retry immidetly.
Before the change, we testes it by delaying the server's startup (and once they started, the request should have been accepted).
Here again, we can argue that the purpose of the test is to see that the client is able to retry (no matter why).
But I also agree that this may be considered as a degenerated test case.
WDYT?